data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -143,13 +134,17 @@ public TestLocalScan(String format) {
           genericRecord.copy(ImmutableMap.of("id", 27L, "data", "overview")),
           genericRecord.copy(ImmutableMap.of("id", 28L, "data", "tender")));
 
-  private void overwriteExistingData() throws IOException {
+  private void overwriteExistingData(Object fileExt) throws IOException {
```

Comment:
why is this change needed? The class itself should be parameterized as I mentioned further above. Then these changes here can be reverted